spec/features/reservations_spec.rb

```diff
@@ -99,7 +99,45 @@
         expect(page).to have_content Time.zone.today.to_s(:long)
         expect(page).to have_content bad_due_date.to_s(:long)
         expect(page).to have_content @eq_model.name
-        expect(page).to have_content 'Create Reservation'
+        expect(page).to have_content 'Confirm Reservation'
+        expect(page).to have_content 'Please be aware of the following errors:'
+        find(:xpath, "//textarea[@id='reservation_notes']").set 'Because'
+        click_button 'Finalize Reservation'
+
+        # check that reservation was created with correct dates
+        query = Reservation.for_eq_model(@eq_model).for_reserver(reserver.id)
+        expect(query.count).to eq(1)
+        expect(query.first.start_date).to eq(Time.zone.today)
+        expect(query.first.due_date).to eq(bad_due_date)
+        expect(query.first.status).to eq('reserved')
+      end
+    end
+
+    shared_examples 'can edit valid reservation' do |reserver|
+      let(:reserver) { reserver }
+      str = reserver ? 'for other user successfully' : 'successfully'
+
+      it str do
+        visit root_path
```

Comment:
I think I edited these existing tests to make sure that it's reaching the new reservation page (and that the new reservation page has all the relevant information). You are correct that these do not test changing the reservation from the new_reservation page. The ones below in the other context perform that role.